src/app/(social-page)/profile/[id]/activities/page.tsx

```diff
@@ -0,0 +1,155 @@
+            </span>
+            <span>Số người tham gia:</span>
+            <span>
+              {event.participants} / {event.maximumParticipants}
+            </span>
+          </p>
+
+          <p className="flex items-center gap-1 text-md">
+            <span>
+              <BarChartFill className="text-[--blue-02]" />
+            </span>
+            <span>Số người tối thiểu: </span>
+            <span>{event.minimumParticipants}</span>
+          </p>
+        </div>
+      </div>
+    </div>
+  )
+}
+
+export default function Page() {
+  const pathname = usePathname()
+  const { replace } = useRouter()
+  const searchParams = useSearchParams()
+  const params = new URLSearchParams(searchParams)
+  const [joinedEvents, setJoinedEvent] = useState([])
+  const [myParams, setMyParams] = useState(`?${params.toString()}`)
+  const [curPage, setCurPage] = useState(Number(params.get('page')) + 1 || 1)
+  const [totalPages, setTotalPages] = useState(1)
+  const onNextPage = () => {
+    if (curPage == totalPages) return
+    params.set('page', curPage.toString())
+    replace(`${pathname}?${params.toString()}`, { scroll: false })
+    setMyParams(`?${params.toString()}`)
+    setCurPage((curPage) => {
+      return curPage + 1
+    })
+  }
+  const onPrevPage = () => {
+    if (curPage == 1) return
+    params.set('page', (curPage - 2).toString())
+    replace(`${pathname}?${params.toString()}`, { scroll: false })
+    setMyParams(`?${params.toString()}`)
+    setCurPage((curPage) => {
+      return curPage - 1
+    })
+  }
+
+  // News list
+  useEffect(() => {
+    axios
+      .get(
+        `${process.env.NEXT_PUBLIC_SERVER_HOST}/events/participated${myParams}&statusId=${POST_STATUS['Bình thường']}`,
+        {
+          headers: {
+            Authorization: `Bearer ${Cookies.get(JWT_COOKIE)}`,
+          },
+        }
+      )
+      .then(({ data: { totalPages, events } }) => {
+        setTotalPages(totalPages)
+        setJoinedEvent(events)
+      })
+      .catch((error) => {})
+  }, [])
+
+  return (
+    <div>
+      <div className="w-full flex flex-col gap-4">
+        <p className="text-[18px] lg:text-[22px] font-bold">
+          Hoạt động đã tham gia
+        </p>
+        <div className="w-full flex flex-col gap-4">
+          {joinedEvents?.length > 0 ? (
+            joinedEvents?.map((event) => (
+              <EventListItem
+                key={event.id}
+                event={event}
+              />
+            ))
+          ) : (
+            <div className="flex items-center gap-2">
+              <Star className="text-[20px] lg:text-[24px]" /> Không có hoạt động
+              đã tham gia để hiển thị
+            </div>
+          )}
+        </div>
+        {/* {totalPages > 1 && (
+        <Pagination
+          totalPages={totalPages}
+          curPage={curPage}
+          onNextPage={onNextPage}
+          onPrevPage={onPrevPage}
+        />
+      )} */}
```

Comment:
```suggestion
        {/* {totalPages > 1 && (
        <Pagination
          totalPages={totalPages}
          curPage={curPage}
          onNextPage={onNextPage}
          onPrevPage={onPrevPage}
        />
      )} */}
```
ko dùng phân trang thì xóa